src/pages/lessons/[slug].tsx

```diff
@@ -268,43 +281,23 @@ const Lesson: React.FC<LessonProps> = ({initialLesson}) => {
         'viewing',
         'completed',
         'addingNote',
-      ].includes(currentPlayerState),
+      ].includes(currentLessonState),
     )
-  }, [currentPlayerState])
+  }, [currentLessonState])
 
   const checkAutoPlay = async () => {
     if (nextLesson) {
       updateResource(nextLesson)
     }
-
     console.debug(`checking autoplay: ${autoplay} [${nextLesson.slug}]`)
-
     if (autoplay && nextLesson) {
       console.debug('autoplaying next lesson', {nextLesson})
       track('autoplaying next video', {
         video: nextLesson.slug,
       })
-
-      if (isFullscreen) {
```

Comment:
As the new player treats fullscreen mode differently, this is no longer necessary.